app/controllers/draft.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const User =require('../models/user');
+const Draft=require('../models/draft');
+
+const response = require('../lib/responses');
+
+
+/**
+ * Adds the Draft to the table
+ * @param  {Express.Request}   	req  - the request object
+ * @param  {Express.Reponse}   	res  - the response object
+ * @param  {Function} 			next - pass to next handler
+ */
+
+function getDraft(req, res, next){
+    //check submitter_id
+	if(!req.body.submitter_id){
+		return response.badRequest(res, `submitter_id is missing`);
+	}
+	//400 bad request if missing either wip_state or submitter_id, or if submitter_id is invalid
+	if (!req.body.wip_state) {
+		return response.badRequest(res, `Draft (wip state) is missing or invalid`);
+	}
+	//TODO wait for middleware
+	User.where({id: req.body.submitter_id}).fetch({require: true})
+		.then(user => {
+			return Draft.forge(req.body).save();
```

Comment:
This will work if the user only provides the expected data, but if they bundle anything else in, this will cause an internal server error because it will try to add data to a column in the database that doesn't exist.

You could either validate there are no additional fields passed in (I do this in several other controllers), or explicitly define what fields go into this e.g.
```
Draft.forge({
    submitter_id: req.user.attributes.id,
    wip_state: req.body.wip_state
}).save();
```